x-pack/plugins/ml/public/application/datavisualizer/file_based/components/import_settings/advanced.tsx

```diff
@@ -123,6 +131,16 @@ export const AdvancedSettings: FC<Props> = ({
         />
       </EuiFormRow>
 
+      <CombinedFieldsForm
+        mappingsString={mappingsString}
+        pipelineString={pipelineString}
+        onMappingsStringChange={onMappingsStringChange}
+        onPipelineStringChange={onPipelineStringChange}
+        combinedFields={combinedFields}
+        onCombinedFieldsChange={onCombinedFieldsChange}
+        results={results}
```

Comment:
`initialized` could be passed in here to disable all buttons when the import is running or has finished.